Tests/Marketplace/marketplace_services.py

```diff
@@ -2218,6 +2224,17 @@ def is_unified_integration(self, file_path: str):
         ])
 
 
+class PackIgnored(object):
+    ROOT_FILES = [Pack.SECRETS_IGNORE, Pack.PACK_IGNORE]
+    NESTED_FILES = {
+        PackFolders.INTEGRATIONS.value: ["README.md", "Pipfile", "Pipfile.lock", "_test.py", "commands.txt"],
+        PackFolders.SCRIPTS.value: ["README.md", "Pipfile", "Pipfile.lock", "_test.py"],
+        PackFolders.TEST_PLAYBOOKS.value: [],
+        PackFolders.PLAYBOOKS.value: ["_README.md"]
+    }
+    NESTED_DIRS = [PackFolders.INTEGRATIONS.value, PackFolders.SCRIPTS.value]
```

Comment:
should we add the `TestPlaybooks` folder too or will the `NESTED_FILES` catch the sub-folders?
![image](https://user-images.githubusercontent.com/30797606/112424340-e7b6c080-8d3c-11eb-8d8a-b503ead133e7.png)

Reply:
The `TestPlaybooks` folder is ignored by the `NESTED_FILES`